src/test/exporter/test_exporter.cc

```diff
@@ -670,27 +670,25 @@ TEST(Exporter, check_labels_and_metric_name) {
   counters_data.emplace_back("ceph-osd.0", "ceph_osd_numpg");
   counters_data.emplace_back("ceph-client.rgw.foo.ceph-node-00.hrgsea.2.94739968030880", "ceph_rgw_get");
 
-  static std::vector<std::pair<labels_t, std::string>> labels_and_name;
-  labels_and_name.emplace_back(labels_t{{"ceph_daemon", "\"osd.0\""}}, "ceph_osd_numpg");
-  labels_and_name.emplace_back(labels_t{{"instance_id", "\"hrgsea\""}}, "ceph_rgw_get");
+  static std::vector<labels_t> labels_and_name;
```

Comment:
There are no names here anymore:
```suggestion
  static std::vector<labels_t> labels_vec;
```